ctrd/container_types.go

```diff
@@ -6,13 +6,23 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
-// Container wraps container's info.
+// Container wraps container's info. there have two kind of containers now:
+// One is created by pouch, no need any different operations,
+// The other is just taken over by pouch, like created by moby, we use `Takeover` flag to mark it,
+// because the container is not created by pouch, so we must specify rootfs the container used, so that
+// we can use the rootfs to create new containerd container.
 type Container struct {
 	ID      string
 	Image   string
 	Runtime string
 	IO      *containerio.IO
 	Spec    *specs.Spec
+
+	// BaseFS is rootfs used by containerd container
+	BaseFS string
+	// Takeover specify if the containerd container is just
+	// taken over by pouch, not created by pouch
+	Takeover bool
```

Comment:
Please change this field. @HusterWan

Reply:
i think we can merge this pr now, and i will create a new pr to support create container by specify rootfs, WDTY?